ruby/test/syskit/test_syskit_plugin.rb

```diff
@@ -196,6 +196,169 @@
+            .use_frames("object" => "object_global", "world" => "wrong_frame")
+            .transformer { frames "object_global", "world_global" }
+
+        e = assert_raises(Transformer::InvalidConfiguration) do
+            syskit_deploy(task_m)
+        end
+        expected_m = "undefined frame wrong_frame"
+        assert_match Regexp.new(expected_m), e.message
+    end
+
+    it "raises if the tasks transform frames arent all assigned" do
+        task_m =
+            data_consumer_m
+            .use_frames("object" => "object_global")
+            .transformer { frames "object_global", "world_global" }
+
+        e = assert_raises(Transformer::MissingFrame) do
+            syskit_deploy(task_m)
+        end
+        expected_m = /could not find a frame assignment for world in/
+        assert_match expected_m, e.message
+    end
+
+    describe "with error capture" do
+        before do
+            @__capture_errors = Syskit.conf.capture_errors_during_network_resolution?
+            Syskit.conf.capture_errors_during_network_resolution = true
+        end
+
+        after do
+            Syskit.conf.capture_errors_during_network_resolution = @__capture_errors
+        end
+
+        it "raises if a chain cannot be resolved" do
+            task_m =
+                data_consumer_m
+                .use_frames("object" => "object_global", "world" => "world_global")
+                .transformer { frames "object_global", "world_global" }
+
+            e = assert_raises(Syskit::NetworkGeneration::PartialNetworkResolution) do
+                syskit_deploy(task_m)
+            end
+
+            invalid_chain = assert_exception_from_partial_network_resolution(
+                e, Transformer::InvalidChain
+            )
+            expected_m = "cannot find a transformation chain to produce object_global => "\
+                         "world_global for DataConsumer.* \\\(task-local frames: object => "\
+                         "world\\\): no transformation from 'object_global' to "\
+                         "'world_global' available"
+            assert_match Regexp.new(expected_m), invalid_chain.message
+        end
+
+        it "raises if at least one of the assigned frames is invalid" do
+            task_m =
+                data_consumer_m
+                .use_frames("object" => "object_global", "world" => "wrong_frame")
+                .transformer { frames "object_global", "world_global" }
+
+            e = assert_raises(Syskit::NetworkGeneration::PartialNetworkResolution) do
+                syskit_deploy(task_m)
+            end
+            invalid_configuration = assert_exception_from_partial_network_resolution(
+                e, Transformer::InvalidConfiguration
+            )
+            expected_m = "undefined frame wrong_frame"
+            assert_match Regexp.new(expected_m), invalid_configuration.message
+        end
+
+        it "raises if the tasks transform frames arent all assigned" do
```

Comment:
```suggestion
        it "raises if the tasks transform frames aren't all assigned" do
```

Reply:
This is a bit too much, not using apostrophe for "arent" is common enough, and even if its not "correct" english per se, it helps when trying to filter unit tests as its easy to omit the `'`